### PR TITLE
Fix Changelog and Readme issues

### PR DESCRIPTION
**Pull request by fmueller**
This PR adds missing changelog entry and fixes broken link in readme.

### CODE REVIEW

File: changelog/Changelog.md [hunk 1]
```diff
@@ -8,5 +8,6 @@ To see a list of all changes, please have a look at the [commit list in Github](
 
 ## Rule Changes
 
+* `2016-12-04:` Changed OAuth flow from implicit to password in [Security section](../security/Security.md).
```
Comment (fmueller) [inline changelog/Changelog.md]: Ok, addressed this in last commit on this branch.

**Comment by fmueller** [comment]
@ePaul @codekultur Can you please check?

**Comment by codekultur** [comment]
👍

**Comment by whiskeysierra** [comment]
:+1:

**Pull request merged**